utils/build-script

```diff
@@ -353,6 +353,21 @@ class BuildScriptInvocation(object):
             args.skip_test_tvos = True
             args.skip_test_watchos = True
 
+        # --build-swift-stdlib is an alias for --build-swift-dynamic-stdlib=1
+        # and --build-swift-static-stdlib=0, which takes precedence over those
+        # options.
+        if args.build_swift_stdlib:
+            args.build_swift_dynamic_stdlib = True
+            args.build_swift_static_stdlib = False
```

Comment:
Perfect, this is just what I need. Thanks, @rintaro!! I'll make the changes first thing tomorrow.